drivers/wifi/eswifi/Kconfig.eswifi

```diff
@@ -30,6 +30,13 @@ config WIFI_ESWIFI_BUS_UART
 
 endchoice
 
+config WIFI_ESWIFI_MAX_DATA_SIZE
+	int "esWiFi message size"
+	default 1600
+	help
```

Comment:
We could have a range here `range xxx yyy`, just don't know what to put as a range.

Reply:
Added range from 500 bytes to 4000 bytes. 3600 bytes covers more than enough my issue